support/chip-testing/src/handler/LegacyControllerCommandHandler.ts

```diff
@@ -549,6 +549,15 @@ export class LegacyControllerCommandHandler extends CommandHandler {
             certSigningRequest,
         );
 
+        if (ca.icacCert !== undefined) {
```

Comment:
I think you can combine that in the below return by just returning "ICAC: ca.icacCert" which becomes undefined and so should be still "not there"